hipify-clang/packaging/hipify-clang.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(hipify-clang)
+
+install(PROGRAMS @HIPIFY_INSTALL_PATH@/hipify-clang DESTINATION bin)
+install(DIRECTORY @HIPIFY_INSTALL_PATH@/include DESTINATION bin)
+
+#############################
+# Packaging steps
+#############################
+set(CPACK_SET_DESTDIR TRUE)
+set(CPACK_INSTALL_PREFIX "/opt/rocm/hip")
+set(CPACK_PACKAGE_NAME "hipify-clang")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Hipify-clang:Clang-based tool to translate CUDA source code into portable HIP C++ automatically")
+set(CPACK_PACKAGE_VENDOR "Advanced Micro Devices, Inc.")
+set(CPACK_PACKAGE_CONTACT "Maneesh Gupta <maneesh.gupta@amd.com>")
+set(CPACK_PACKAGE_VERSION @HIP_VERSION_MAJOR@.@HIP_VERSION_MINOR@.@HIP_VERSION_PATCH@)
+set(CPACK_PACKAGE_VERSION_MAJOR @HIP_VERSION_MAJOR@)
+set(CPACK_PACKAGE_VERSION_MINOR @HIP_VERSION_MINOR@)
+set(CPACK_PACKAGE_VERSION_PATCH @HIP_VERSION_PATCH@)
```

Comment:
I'd prefer setting HIP version to hipify-clang conditionally. If we are called from HIP cmake building process (HIP's cmake variable `BIN_INSTALL_DIR` is set, for instance), then let it be HIP version, otherwise (standalone build) - LLVM version.

Reply:
I tried out BIN_INSTALL_DIR variable but it seems like it cannot be accessed. I've updated you the specifics over on email.